Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
+  sha256 "79ce7ead4ccb159b84bbab05e8a3e569c3b98b032ee58bdfd13df25acc85d5a2"
+
+  depends_on "libpng"
+
+  def install
+    cd "src"
+    system "make"
+    bin.install "compare_pngs"
+  end
+
+  test do
+    # unfortunatley, butteraugli fails when applied to the test fixtures
+    # system "#{bin}/compare_pngs", test_fixtures("test.png"), test_fixtures("test.png")
```

Comment:
If that fails, that's a strong indication that something is broken with the build (though there might be other reasons). Can you provide more context on how exactly this test is failing for you? Ultimately, we'll want to have a test that is both non-trivial and passes successfully.